### PR TITLE
Update all dependent gems.

### PR DESCRIPTION
**Pull request by dazoakley**
This includes several critical security patches (for nokogiri/libxml),
and also updates to sinatra 2.0.0 - hence the changes to the regex's in
the service class.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -2,6 +2,18 @@ Metrics/LineLength:
...
+    - describe
+    - context
+
+Metrics/MethodLength:
+  CountComments: false
+  Max: 20
+
+Metrics/AbcSize:
+  Max: 40
+
```
Comment (baob) [inline .rubocop.yml]: Where do these changes come from ? Are they manual tweaks by you to get the existing immagine code past a new set of rubocop algorithms/defaults ?

Comment (baob) [inline .rubocop.yml]: Or do they come indirectly via mac-utils ?

**Comment by dazoakley** [comment]
Via mac-utils

**Comment by baob** [comment]
:+1:

**Pull request merged**